app/frontend/src/features/Contribute.tsx

```diff
@@ -0,0 +1,98 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import { LOGIN } from "features/auth/constants";
+import DesktopAuthBg from "features/auth/resources/desktop-auth-bg.jpg";
+import { Link } from "react-router-dom";
+import { loginRoute } from "routes";
+
+import { COUCHERS } from "../constants";
+import ContributorForm, { CONTRIBUTE, JOIN_THE_TEAM } from "./contributorForm";
+
+const useStyles = makeStyles((theme) => ({
+  bg: {
+    height: "100%",
+    width: "100%",
+    backgroundColor: theme.palette.common.white,
+    // backgroundAttachment: "scroll, local",
```

Comment:
```suggestion
    // backgroundAttachment: "scroll, local",
```
Remove if we don't need this?